src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+    private static final Lottery LOTTERY = new Lottery();
```

Comment:
It may look like a constant because we initialize it only once here, but this is essentially a service that is usually initialized in the constructor and created somewhere outside of this class to not hardcode them (google `Dependency injection` and `Inversion of control` if you are curious but do not dig too deep since it is covered in details in the course)

So it can not be named as a constant, usual `lottery` would be fine, and lets not make it static, to follow-up with the OOP principles :)